packages/pn-commons/src/services/tracking.service.ts

```diff
@@ -30,16 +30,33 @@ export function trackEvent(event_name: string, nodeEnv: string, properties?: any
 /**
  * Set profile properties
  */
-export function setProfileProperty(property: any, nodeEnv: string): void {
+export function setProfileProperty(
```

Comment:
let's change this method name from setProfileProperty to setSuperOrProfileProperty